ui/src/main/java/com/midtrans/sdk/ui/views/transaction/TransactionActivity.java

```diff
@@ -235,11 +238,33 @@ private void showPaymentActivity(PaymentMethodModel paymentMethod) {
             case PaymentType.GCI:
                 startGiftCardPaymentFlow();
                 break;
+            case PaymentType.TELKOMSEL_CASH:
+                startTelkomselCashFlow();
+                break;
+            case PaymentType.XL_TUNAI:
+                startXlTunaiFlow();
+                break;
             default:
                 break;
         }
     }
 
+    private void startXlTunaiFlow() {
```

Comment:
Please rename to `startXlTunaiPaymentFlow` and `startTelkomselCashPaymentFlow` for more consistent naming.

Also please sort the function name based on when it's called.

`startTelkomselCashFlow` and `startXlTunaiFlow` should be after `startGiftCardPaymentFlow`